framework/tlvf/tlvf.py

```diff
@@ -386,7 +386,7 @@ def __init__(self, src_path, yaml_path, out_path, conf_path, print_dependencies,
             sys.exit(0)
 
         self.loadAllYamlFilesToDB()
-        self.copyIncludeSource()
+        #self.copyIncludeSource()
```

Comment:
Don't comment it out, just remove it. And remove the definition of copyIncludeSource as well, and recursively anything else that is no longer needed.